eslint.config.mjs

```diff
@@ -0,0 +1,26 @@
+// @ts-check
+import { createConfigForNuxt } from '@nuxt/eslint-config/flat'
+
+export default createConfigForNuxt({
+  features: {
+    stylistic: true,
+    tooling: true,
+  },
+  dirs: {
+    src: ['src'],
+  },
+})
+  .append({
+    ignores: ['playground', 'docs', 'test'],
+  })
+  .override('nuxt/typescript/rules', {
+    rules: {
+      // TODO: Discuss if we want to enable this
+      '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
The typing around the Leaflet ecosystem isn't the best at the moment, which result in such things being somehow required :

```typescript
declare module 'leaflet.markercluster' {
  export const MarkerClusterGroup: any;
}
```
```typescript
import { ref } from 'vue';
const map = ref(null) as any; // So we can call => map.value.leafletObject
```

So I agree with turning that rule off for now.
Maybe we should comment the line with something like "Required while typing around Leaflet's ecosystem isn't cleaned up." ?

Reply:
I think the same way. In the end, it would be healthier to take it in the direction of removal.

> So I agree with turning that rule off for now.

The scope of application of this ESLint rule is the developer of this module.
Therefore, I think there is a way to write it in README.md so that developers can understand it.

> Maybe we should comment the line with something like "Required while typing around Leaflet's ecosystem isn't cleaned up." ?

Reply:
Maybe it make sense to add some sort of `CONTRIBUTING.md` file to make it clear + add basic contribution stuff ? (Something like https://github.com/nuxt-modules/i18n/blob/main/CONTRIBUTING.md I guess)

I prefer not to overload the README file if possible (I mean I hate it when I open a git repo and the Readme is 1000 lines lol)

Reply:
Which mean we could remove the "Development" section of the README.md and make a link to CONTRIBUTING.md at the top of the README.

Reply:
Well, the contribution guide would be better since it's for development.
Since the content is different from this PR, it would like to create a separate issue and create a contribution guide.

Reply:
Btw, a contribution guide was recently added to Nuxt Script. This may be helpful.

https://github.com/nuxt/scripts/pull/109
https://scripts.nuxt.com/docs/getting-started/contributing

Reply:
Yeah you can create a separate issue ! 😄
The contributing guide could be in the "About" section of the doc, should look better.

For this PR just keep a comment on the file and I'll merge.

Reply:
@shinGangan Can you update the comment with "Required while Leaflet's typing isn't improved" ?

Reply:
@Gugustinette 

Sorry, I overlooked it...

I'll respond to added the comments!
Also, there's a conflict, so I'll address that as well.